packages/iTwinUI-react/src/core/Wizard/Stepper.tsx

```diff
@@ -32,7 +32,7 @@ export type StepperProps = {
    */
   steps: StepProperties[];
   /**
-   *  The type of Wizard to display.
+   *  The type of Stepper to display.
```

Comment:
You didn't change the name of the parent file from 'Wizard' to 'Stepper'.

Reply:
Yes, I just deprecated Wizard but left `Wizard.tsx` as it was because of the following:

![image](https://user-images.githubusercontent.com/45748283/200007207-bc2f3239-f600-4079-a604-62ce64062007.png)

Also, Stepper actually already existed, but it was just being wrapped with Wizard ([L59-70](https://github.com/iTwin/iTwinUI-react/pull/905/files#diff-31a31bcd2c19607b8a9dfdfdd6da3ac2990b7dfd28d59ec85314055793880fa8R59-R70)).

Or maybe I misunderstood what you meant?

Reply:
Just to mention, I changed the highlighted L35 because Wizard is now deprecated. So I thought the JSDocs in `Stepper.tsx` should also not use the word "Wizard".

But `Stepper.tsx` is a file that already existed and I left it unchanged apart from L35.

Reply:
I meant 
![image](https://user-images.githubusercontent.com/37936169/200060256-4710dd40-a2e1-4ca2-9b8c-18ee694aceb4.png)

Reply:
Ah, nice catch! Renamed `src/core/Wizard` to `src/core/Stepper`